cobbler/cobbler_collections/manager.py

```diff
@@ -184,9 +201,11 @@ def deserialize(self):
             try:
                 serializer.deserialize(collection)
             except Exception as e:
-                raise CX("serializer: error loading collection %s: %s. Check /etc/cobbler/modules.conf" % (collection.collection_type(), e))
+                raise CX("serializer: error loading collection %s: %s. Check /etc/cobbler/modules.conf"
+                         % (collection.collection_type(), e))
```

Comment:
```suggestion
                raise CX("serializer: error loading collection %s: %s. Check /etc/cobbler/modules.conf"
                         % (collection.collection_type(), e)) from e
```
This should satisfy Codacy.